### ISSUE TITLE
RCS1163 false positive for discard parameter _1, _2, ...

### ISSUE THREAD

**Issue by jbradt-stiles** [JosefPihrt/Roslynator#770]
**Product and Version Used**: roslynator 2019 3.1.0

**Steps to Reproduce**:
1) create a method with multiple discard parameters (_, _1)
2) analyze solution

**Actual Behavior**:
```csharp
public static bool DoSomething(
            int _,
            int _1
        ) =>
            true;
```
raises RCS1163: Unused parameter for _1 which, from what I can see, should be a valid format for a discard parameter

logic for roslyn analyzer for IDE0060 is [here](https://github.com/dotnet/roslyn/blob/387082686aad21a0fe091adf30e7527c02d2f0f4/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ISymbolExtensions.cs#L749)
````csharp
public static bool IsSymbolWithSpecialDiscardName(this ISymbol symbol)
            => symbol.Name.StartsWith("_") &&
               (symbol.Name.Length == 1 || uint.TryParse(symbol.Name.Substring(1), out _));
````

**Expected Behavior**:

don't raise RCS1163 for the format _x where x is a number

**Comment by JosefPihrt** [comment]
Sounds reasonable!
